x/dclupgrade/client/cli/query_approved_upgrade.go

```diff
@@ -0,0 +1,73 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/dclupgrade/types"
+)
+
+func CmdListApprovedUpgrade() *cobra.Command {
```

Comment:
Please correct the generated command to align with `CmdListProposedUpgrade`
(check also https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/README-DEV.md#how-to-add-a-new-module)